tests/test_unit_registry.py

```diff
@@ -0,0 +1,15 @@
+import ansys.units as ansunits
+
+
+def test_default_units():
```

Comment:
You should add a test to configure a custom registry

Reply:
I have added the test.